server/controllers/article.controller.js

```diff
@@ -149,22 +149,31 @@ class ArticleController {
      * @returns {object}
      * @memberof getAllArticles
     */
+    const { page } = req.params;
+    const limit = 10;
+    const offset = limit * (page - 1);
+    const order = [['updatedAt', 'DESC']];
```

Comment:
`createdAt` should be used to sort the articles so the order of articles will be fixed

Reply:
> `createdAt` should be used to sort the articles so the order of articles will be fixed

I actually put createdAt at first, changed it while testing. Will effect the change. Thanks

Reply:
I do not think so but using `updatedAt` means once a (user) author updates an article it listing on the categories search result will change because `updatedAt` value is bound to change anytime the user edit or change something in the article.

Reply:
@geekdanitek nice observation 👍

Reply:
@cupidcoder Is this your review?

Reply:
> @cupidcoder please I don't understand your review.

@BukkyOmo Apologies if I didn't communicate correctly. I made a comment and not a review, on Daniel's observation below.

> `createdAt` should be used to sort the articles so the order of articles will be fixed